src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
```

Comment:
There is an unnecessary empty line at the start of the class definition. Please remove it to follow best practices.